### DISCUSSION TITLE
如何实现多次function calling

### DISCUSSION THREAD

**Discussion by wang-qj** [zai-org/ChatGLM3#1331] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ff2efdb4-f914-47ef-afdc-499410d63efc)
看介绍应该是支持多次function calling的，但实际测试下来只能回答一次function calling

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个版本的模型多次fc能力很差，建议升级到 GLM-4-Air Plus等模型能支持多个fc，GLM3-6b，GLM-4-9b-Chat均不具备完整的多个工具调用的原生能力。
